src/useObserver.ts

```diff
@@ -34,7 +35,15 @@ export function useObserver<T>(fn: () => T, baseComponentName = "observed"): T {
         }
     })
     if (exception) {
+        reaction.current.dispose()
         throw exception // re-throw any exceptions catched during rendering
     }
     return rendering
 }
+
+function printDebugValue(v: React.MutableRefObject<Reaction | null>) {
```

Comment:
This is causing an annoying drop in the coverage. I would suggest separating it extra file and have some simple test for it. I doubt it can be tested from the component.

Reply:
Will do

Reply:
Done